chat-agent/tests/test_content_moderation.py

```
@@ -19,812 +19,389 @@
+            assert is_flagged is False
+            assert category == ""
+
+    async def test_returns_empty_category_for_safe_content(self):
+        """Test that empty string category is returned for safe content."""
+        mock_response = _make_moderation_response(
+            flagged=False,
+            categories={"hate": False, "violence": False}
         )
+        with patch("api.http.moderation_client") as mock_client:
+            mock_client.moderations.create = AsyncMock(return_value=mock_response)
+            is_flagged, category = await check_moderation("Hello, world!")
```

Ruff (0.15.12)

[warning] Unpacked variable `is_flagged` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)